lib/client-assets.php

```diff
@@ -752,6 +752,11 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	), $meta_box_url );
 	wp_localize_script( 'wp-editor', '_wpMetaBoxUrl', $meta_box_url );
 
+	// Ensure that we've got jQuery.
+	if ( ! wp_script_is( 'jquery', 'done' ) ) {
+		wp_enqueue_script( 'jquery' );
+	}
```

Comment:
Why can't we just add a `jquery` dependency, something like this https://github.com/WordPress/gutenberg/pull/3345/commits/255ef193d1c0a0bd30e1863dc89da4875f799b86

Reply:
We can only pass dependencies into `wp_enqueue_script`, not `wp_add_inline_script`: https://developer.wordpress.org/reference/functions/wp_add_inline_script/

Reply:
But `wp_add_inline_script` is tied to the original handle `wp-editor`, and we can define jQuery as a dependency for the enqueuing of `wp-editor`.